samples/sample_encode/src/sample_encode.cpp

```diff
@@ -305,6 +333,10 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
                 return MFX_ERR_UNSUPPORTED;
             }
         }
+		else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-msb10")))
```

Comment:
alignment is wrong

Reply:
it is correct - web viewer displays tabs incorrectly